lib/puppet/face/man.rb

```diff
@@ -39,12 +39,28 @@
     EOT
 
     default
-    when_invoked do |name, options|
-      if legacy_applications.include? name then
-        return Puppet::Application[name].help
+    when_invoked do |*args|
+      # [<argument> ...] <options>
+      options = args.pop
```

Comment:
Are options used anywhere, or is this just to remove it from `args` so it doesn't interfere with anything?

Reply:
The options hash is always passed in last, and would contain any command line options specific to the man face. There aren't any so it should always be empty, but I wanted to take it off and show that its always there.

Reply:
ok